llama_stack/core/datatypes.py

```diff
@@ -120,10 +120,9 @@ class AutoRoutedProviderSpec(ProviderSpec):
     provider_data_validator: str | None = Field(
         default=None,
     )
-
-    @property
-    def pip_packages(self) -> list[str]:
-        raise AssertionError("Should not be called on AutoRoutedProviderSpec")
+    pip_packages: list[str] = Field(
```

Comment:
this, seems like ... a separate issue but one I had to wrangle with to get this working.

Reply:
actually, let me try just removing this?